low-level/framework/base/sspl_constants.py

```diff
@@ -109,6 +109,33 @@
 
 SSPL_CONFIGS = ['log_level', 'cli_type', 'sspl_log_file_path', 'cluster_id', 'storage_enclosure', 'setup', 'operating_system']
 
+MODULES = {"CPUFAULTSENSOR": {"probe" : "sysfs", "threaded": "true"},
```

Comment:
May have conflicting changes with following, address this PR after that if seen conflicting. Changes here looks more cleaner though
https://github.com/Seagate/cortx-monitor/pull/167/files
low-level/framework/base/sspl_constants.py

Reply:
@cdeshmukh, Sure I will check for the possible conflicts and will address this after this PR (https://github.com/Seagate/cortx-monitor/pull/167/files) has merged.

Reply:
@cdeshmukh, @thavanathan Resolved Conflicts and updated patch with latest code.